src/test/components/StackChart.test.js

```diff
@@ -21,8 +26,7 @@ import {
   removeRootOverlayElement,
 } from '../fixtures/utils';
 import { getProfileFromTextSamples } from '../fixtures/profiles/processed-profile';
-import { changeSelectedCallNode } from '../../actions/profile-view';
-import { selectedThreadSelectors } from '../../selectors/per-thread';
```

Comment:
I did a bit of reordering in the imports in some tests. I like this order, with empty lines in-between:

1. 3rd party libraries
2. our app code
3. our test code

Especially when it's a bit busy with a lot of imports.